kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/BluetoothConnector.java

```diff
@@ -20,8 +20,13 @@
 import javax.microedition.io.Connection;
 import javax.microedition.io.ConnectionNotFoundException;
 
+import org.osgi.annotation.versioning.ProviderType;
 import org.osgi.service.io.ConnectorService;
 
+/**
+ * @noimplement
+ */
```

Comment:
It would be better if you can use the full version of @noimplement with full sentence.